fvm/src/machine/default.rs

```diff
@@ -196,41 +203,84 @@ where
     }
 
     fn transfer(&mut self, from: ActorID, to: ActorID, value: &TokenAmount) -> Result<()> {
-        if from == to || value.is_zero() {
-            return Ok(());
-        }
-
-        if value.is_negative() {
-            return Err(syscall_error!(IllegalArgument;
+        if self.context.network_version >= V15 {
```

Comment:
`NetworkVersion::V15` is easier to understand.

Reply:
yessir